lcoe_calculator/base_processor.py

```diff
@@ -126,7 +126,7 @@ def __init__(
 
```

Comment:
I missed it in a past review, but `tcc` is missing from the `__init__()` docstring